### PR TITLE
chore: lint fixes + doc update

### PR DESCRIPTION
**Pull request by thelostone-mc**
#### Description 

- updates doc + .env template to make it clearer on what Infura to use
- updates doc to replace `yarn compile` with `yarn build` (as the former does not exist )
- fixes all lint issues that popup within the contract 
- fixes almost lint issues that show within the contract/tests (2 lint rules continue to fail )

**Before:** 
- [Linting Errors in Contract](https://user-images.githubusercontent.com/5358146/126417929-8148365a-8651-4c9d-a31e-9b207cb4d4e5.png)
- [Linting Errors in Contract Tests](https://user-images.githubusercontent.com/5358146/126418024-10d040b8-d0fb-43f2-8c34-03d531fbd9a2.png)

**After:** 
- [SEE HERE](https://user-images.githubusercontent.com/5358146/126422755-64e16ba3-ee2b-4d8b-b75b-d1610198a295.png)

We have errors in `timeTravel` and I wasn't able to figure out what Provider type to pass in there. Once that is fixed -> linting will pass completely 

#### Testing 

```
# Made lint changes in contract
cd contracts
yarn clean
yarn build
cd ..
yarn dev 
# App booted without any issues
```
Screenshot 2021-07-21 at 7 17 07 AM

#### Tests Pass

Screenshot 2021-07-21 at 7 34 11 AM

### CODE REVIEW

File: contracts/.env.template [hunk 1]
```diff
@@ -1,2 +1,2 @@
-INFURA_ID=zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz
+INFURA_ID=yourInfuraApiKey
```
Comment (wildmolasses) [inline contracts/.env.template:1]: there was some confusion earlier about "infura api key" vs "infura project id". I might recommend:

```suggestion
INFURA_ID=yourInfuraProjectId
```

File: contracts/README.md [hunk 2]
```diff
@@ -73,3 +75,11 @@ Delete the smart contract artifacts, the coverage reports and the Hardhat cache:
 ```sh
 $ yarn clean
 ```
+
+
+### Hardhat console
+
+To access the console once the hardnat network is running on your local machine.
+```sh
+$ npx hardhat console
```
Comment (wildmolasses) [inline contracts/README.md:84]:
```suggestion
$ yarn hardhat console
```

**Review by wildmolasses — COMMENTED** [review]
a couple nits, otherwise lgtm. hmm, wonder why CI isn't running. maybe after rebase it will.

**Review by phutchins — APPROVED** [review]
Looks good!

**Review by mds1 — APPROVED** [review]

**Pull request merged**